docs/UserGuide.adoc

```diff
@@ -234,19 +240,19 @@ There is no need to save manually.
 
 == Command Summary
 
-* *Add* `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` +
+* *Add* `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` (`a n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` +
 e.g. `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
-* *Clear* : `clear`
-* *Delete* : `delete INDEX` +
+* *Clear* : `clear` (`clr`)
+* *Delete* : `delete INDEX` (`del INDEX`)+
 e.g. `delete 3`
 * *Edit* : `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]...` +
 e.g. `edit 2 n/James Lee e/jameslee@example.com`
-* *Find* : `find KEYWORD [MORE_KEYWORDS]` +
+* *Find* : `find KEYWORD [MORE_KEYWORDS]` (`f KEYWORD [MORE_KEYWORDS]`) +
 e.g. `find James Jake`
-* *List* : `list`
+* *List* : `list` (`ls`)
 * *Help* : `help`
 * *Select* : `select INDEX` +
 e.g.`select 2`
-* *History* : `history`
+* *History* : `history` (`h`)
```

Comment:
I think h is more valid for help. History is not as important, so it can have another alias or none at all.

Reply:
I actually think history might be more commonly used. Help is only used when a new user is starting out on addressbook. History would probably of more use to a seasoned user.